index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<section class="toolbar">
+  <nav class="toolbar logo"><img src="images/logo.png" alt=""></nav>
```

Comment:
- [ ] you should only use one `<nav>` tag in your file so please try to remove that one and leave just one that should wrap the `menu icon `